values.yaml

```diff
@@ -77,3 +77,74 @@ storage:
   elasticsearch:
     port: 9200
     host: 127.0.0.1
+
+exporter:
+  enabled: false
+  image:
+    repository: registry.ng.bluemix.net/secadvisor_dev/skydive-flow-exporter
```

Comment:
@sa-cloud - you must use a the standard docker hub (not an IBM repo)

Reply:
We still wait for the official image to be built and placed in the standard location. Meanwhile I can place a temporal image, built by @dubek in a temporal location